libc/test/src/math/exhaustive/CMakeLists.txt

```diff
@@ -142,6 +142,21 @@ add_fp_unittest(
     -lpthread
 )
 
+add_fp_unittest(
+  exp2m1f_test
+  NO_RUN_POSTBUILD
+  NEED_MPFR
+  SUITE
+    libc_math_exhaustive_tests
+  SRCS
+    exp2m1f_test.cpp
+  DEPENDS
+    .exhaustive_test
+    libc.src.math.exp2m1f
```

Comment:
Why is `exp2f_test` above depending on `libc.include.math` and `libc.src.__support.FPUtil.fp_bits`?

Reply:
It's an old test, don't worry about that / update it if you want to.

Reply:
I looked at some of the other surrounding tests and they also seem to unnecessarily depend on `libc.src.__support.FPUtil.fp_bits`. I won't try to find and update them all now.